android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/performance/SyncPerformanceTestUtils.kt

```diff
@@ -94,6 +106,70 @@ class SyncPerformanceTestUtils {
                 preserveRawOutputProp, defaultPreserveRawOutput)!!.toBoolean()
         }
 
+        internal fun asertLocalAndRemoteDBCount(ctx: SyncPerformanceTestContext, numDocs: Int) {
+            // Verify that the test did indeed synchronize the updates remotely
+            val numSyncedIds = Tasks.await(ctx.testColl.sync().syncedIds).size
+            val numLocalDocs = Tasks.await(ctx.testColl.sync().count()).toInt()
+            val numRemoteDocs = Tasks.await(ctx.testColl.count()).toInt()
+
+            SyncPerformanceTestUtils.assertIntsAreEqualOrThrow(
+                numSyncedIds, numDocs, "Number of Synced Ids")
+            SyncPerformanceTestUtils.assertIntsAreEqualOrThrow(
+                numLocalDocs, numDocs, "Number of Local Documents")
+            SyncPerformanceTestUtils.assertIntsAreEqualOrThrow(
+                numRemoteDocs, numDocs, "Number of Remote Documents")
+        }
+
+        internal fun assertIntsAreEqualOrThrow(actual: Int, expected: Int, message: String = "") {
+            if (expected == actual) {
+                return
+            }
+            throw Exception(String.format("Expected: %s but found %s: %s", expected, actual, message))
```

Comment:
[nit] We might want to take advantage of existing testing libraries for our failures instead of throwing general exceptions

Reply:
+1, or we should at least define our own `StitchPerformanceTestException` or something along those lines